library/pk.c

```diff
@@ -232,7 +240,11 @@ int mbedtls_pk_verify_restartable( mbedtls_pk_context *ctx,
                const unsigned char *sig, size_t sig_len,
```

Comment:
Non-blocker: Consider fixing the indentation while at it.

Reply:
I prefer not to change unrelated things. And by the way as a reviewer I prefer when unrelated things don't change.